saleor/graphql/order/mutations/order_cancel.py

```diff
@@ -37,7 +36,6 @@ class Meta:
         error_type_field = "order_errors"
 
     @classmethod
-    @traced_atomic_transaction()
```

Comment:
We still need transactions from lines 46 to 52.

Reply:
true. Fixed it.